storybookjs/storybook
# CLI/Vue: add interactions to vue cli template

Issue: https://github.com/storybookjs/storybook/issues/16722

## What I did
Similar to PR https://github.com/storybookjs/storybook/pull/17345. Added interactions to Vue cli template.

## How to test
Build cli locally and run repro to select Vue as a framework

If your answer is yes to any of these, please make sure to include it in your PR.
